contrib/cni/README.md

```diff
@@ -63,7 +63,7 @@ Download the `CNI` plugins source tree:
 ```bash
 git clone https://github.com/containernetworking/plugins
 cd plugins
-git checkout v1.1.1
+git checkout main
```

Comment:
This is probably fine, provided that things on `main` never introduce breaking changes or break backward compatibility.